close_jira.py

```diff
@@ -0,0 +1,78 @@
+from jira import ( JIRA,JIRAError )
```

Comment:
is this file used? or do we plan to use in the future?

Reply:
removing this file as closing JIRA requires more thought process